### PR TITLE
Add prominence_icon to admin user listings

### PR DESCRIPTION
**Pull request by garethrees**
Add prominence_icon to admin user listings

Allows an at-a-glance look at whether a User is banned, inactive or
otherwise in a compact method. When banned a user gets “Account
Suspended” prepended to their name, and in some cases an orange “banned”
label, but in other more information-dense listings we either don’t show
this or it’s hard to parse out.

This makes it easy to tie banned users to hidden content or hidden
content to unbanned users.

Main benefit is in a listing like this where we can get a quick overview of a group of content:

Screenshot 2022-02-24 at 17 32 08

Also tweaks the users index. Not much more information than before, but marginally less messy.

Screenshot 2022-02-24 at 17 31 07

### CODE REVIEW

**Comment by WilliamWDTK** [comment]
>When banned a user gets “Account Suspended” prepended to their name,

Isn't it appended/postpended?

**Comment by garethrees** [comment]
> Isn't it appended/postpended?

🤦 yes, oops.
